node/config/types.go

```diff
@@ -691,6 +691,10 @@ type FevmConfig struct {
 }
 
 type Events struct {
+	// Save specifies whether or not to save emitted events, or discard them. Events are
+	// discarded by default.
+	Save bool
```

Comment:
Name sounds good

We should make the comment say that it is enabled implicitly when EnableEthRPC is set - or we can make the node when save isn't set when EnableEthRPC is (can be done easily with the `Error` node builder option)